src/lib/services/workspaces/update.ts

```diff
@@ -0,0 +1,13 @@
+import { db } from '@/lib/db'
+import { z } from 'zod'
+
+export const updateWorkspaceInput = z.object({
+  id: z.string(),
+  name: z.string().optional(),
+  description: z.string().optional(),
+})
```

Comment:
Same idea as my other comment, I think this can be moved to `src/lib/schemas/update-note.ts`